### PR TITLE
mcservman: Add version 0.0.6

### PR DESCRIPTION
**Pull request by CrazyWillBear**

### CODE REVIEW

**Comment by CrazyWillBear** [comment]
If you want to make sure this isn't a virus, the project's source code and such is available [here](https://github.com/CrazyWillBear/MinecraftServerManager)

**Review by Calinou — CHANGES_REQUESTED** [review]

**Comment by CrazyWillBear** [comment]
@Calinou I just comitted your changes, thanks!

**Comment by CrazyWillBear** [comment]
@Calinou is all good now with the pull request?

**Comment by Calinou** [comment]
The download URL returns `404 Not Found`: https://github.com/CrazyWillBear/MinecraftServerManager/releases/download/v0.0.6-alpha/v0.0.6-alpha.zip

**Comment by CrazyWillBear** [comment]
@Calinou Oh crap, that's an error with the GitHub release not the json. I just fixed it, it should work now, sorry!

**Comment by CrazyWillBear** [comment]
@Calinou just fixed the error with the GitHub link. It should work now, I tested it on my computer and so far so good.

**Comment by CrazyWillBear** [comment]
@Calinou OMG I made another mistake, I just fixed it. Now FOR SURE it will work, I had other friends test it. Sorry!

File: bucket/mcservman.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.0.6",
+    "description": "A commandline Minecraft server manager",
+    "homepage": "https://github.com/CrazyWillBear/MinecraftServerManager",
+    "license": "MIT",
+    "suggest": {
+        "Java Runtime Environment": "java/adoptopenjdk-hotspot-jre"
+    },
+    "url": "https://github.com/CrazyWillBear/MinecraftServerManager/releases/download/v0.0.6-alpha/v0.0.6-alpha.zip#/mcservman.zip",
```
Comment (Calinou) [inline bucket/mcservman.json:9]:
```suggestion
    "url": "https://github.com/CrazyWillBear/MinecraftServerManager/releases/download/v0.0.6-alpha/v0.0.6-alpha.zip#/mcservman.zip",
    "hash": "917de83fca4a193d9d03828145c7f9cc7cb1a5a04e1e5d906df4248a9263f941",
```

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by CrazyWillBear** [comment]
Thanks so much!!!!

**Comment by CrazyWillBear** [comment]
@Calinou one last thing, I'm sorry: I'm kind of new to GitHub (if you can't tell), and the bucket/mcservman.json for me isn't showing up in the repository for me. Is this an error? Do I have to do something? So sorry and this is the final message I'll be burdening you with.

**Comment by Calinou** [comment]
I tested the app locally and it's not running out of the box:

```
A fatal error occurred. The required library hostfxr.dll could not be found.
If this is a self-contained application, that library should exist in [C:\Users\Hugo\scoop\apps\mcservman\current\].
If this is a framework-dependent application, install the runtime in the global location [C:\Program Files\dotnet] or use the DOTNET_ROOT environment variable to specify the runtime location or register the runtime location in [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x64\InstallLocation].

The .NET runtime can be found at:
  - https://aka.ms/dotnet-core-applaunch?missing_runtime=true&arch=x64&rid=win10-x64&apphost_version=5.0.5
```

Did you forget to bundle something in the release?

**Comment by CrazyWillBear** [comment]
Oh that's not an issue: the app requires Dotnet core runtime to be installed, but I'm going to add a check for that in the software itself later. No need to worry!

**Pull request merged**